missions/views/mission.py

```diff
@@ -25,6 +25,16 @@ class DetailView(LoginRequiredMixin, PermissionRequiredMixin, generic.DetailView
     nav_item = 'missions'
     permission_required = 'missions.view_mission'
 
+    def get_context_data(self, **kwargs):
+        mission = self.get_object()
+        multiple_assigned_users = list()
+
+        for assignment in mission.assignment_set.all():
+            intersection = [value for value in assignment.user.assignments.all() if value in mission.assignment_set.all()]
```

Comment:
This don't work if there are assignments that do not yet have a user. `assignment.user` will be `None`.

Reply:
should work now